v2/cli/init_templates.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for Brigade projects
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  ## Subscribe below to any events that should trigger your script.
+  ## The commented example depicts a subscription to "exec"
+  ## events originating from the Brigade CLI. 
+  # eventSubscriptions:
+  # - source: brigade.sh/cli
+  #   types:
+  #   - exec
+  workerTemplate:
+    # logLevel: DEBUG
+    {{- if eq .GitCloneURL "" }}
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{- nindent 8 .Script }}
+    {{- else }}
+    git:
```

Comment:
Let's add a comment above this line that indicates that these `git` details can be overridden by individual events. This will explain away behavior that might otherwise surprise people.

Something like this:

```
## Your brigade.{{ .Language }} script will be loaded from the following git repository by default, but note that individual events CAN override these details. The Brigade GitHub Gateway, for instance, overrides these details for relevant events so the events themselves reference applicable pull branches, tags, pull requests, etc.
```